packages/trader/src/Constants/contract.tsx

```diff
@@ -279,15 +288,15 @@ export const getSupportedContracts = is_high_low => ({
     },
 });
 
-export const getContractConfig = is_high_low => ({
+export const getContractConfig = (is_high_low: boolean) => ({
     ...getSupportedContracts(is_high_low),
     ...getUnsupportedContracts(),
 });
 
-export const getContractTypeDisplay = (type, is_high_low = false, show_button_name = false) => {
-    const contract_config = getContractConfig(is_high_low)[type];
-    return (show_button_name && contract_config.button_name) || contract_config.name || '';
+export const getContractTypeDisplay = (type: string, is_high_low = false, show_button_name = false) => {
```

Comment:
```suggestion
export const getContractTypeDisplay = (type: string, is_high_low:boolean = false, show_button_name:boolean = false) => {
```
Do we need the type checking here?
Just a question

Reply:
There are no errors (or other underlines from TS) as it can determine types of `is_high_low` and `show_button_name`, because they have default boolean values. As I checked in our code base, it can be used like I wrote above (example is in packages/shared/src/utils/constants/contract.tsx line 472).
But if it's considered as a bad approach, i can change.